images/linux/Ubuntu2004-README.md

```diff
@@ -128,6 +128,12 @@
 | Composer | 2.0.13       |
 | PHPUnit  | 8.5.15       |
 
+Both xdebug and pcov are installed but only xdebug is enabled. Run these commands to switch to pcov:
```

Comment:
Docs are generated dynamically. 
Please use the scripts here to update the docs.
https://github.com/actions/virtual-environments/tree/main/images/linux/scripts/SoftwareReport

Reply:
@shivammathur I don't know these scripts language so I don't know how to do it.

Can you please do it yourself?

Reply:
@sylfabre
Done, 
Remove the changes in the README and add this commit https://github.com/shivammathur/virtual-environments/commit/fa3fcb9fa81708a9241d45593901f0686b6fa4b7 to your PR.

Reply:
Done

Reply:
@sylfabre Please also update the `images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1` file to call `Build-PHPSection`. Refer to the second diff in the commit https://github.com/shivammathur/virtual-environments/commit/fa3fcb9fa81708a9241d45593901f0686b6fa4b7

Reply:
@shivammathur Done, sorry for this oversight 🙈